01_temperature_processing.R

```diff
@@ -44,7 +45,7 @@ library(plyr)
 
 #Set the data file path and file that you want to process
 src_file <- '//deqlead02/Vol_Data/salmon-drift/2015/Fake2SDCWC2015CDO.xls'
-SubID <- 'XXXX' # Enter the submission ID from VolWQDB
+SubID <- '0020' # Enter the submission ID from VolWQDB
```

Comment:
Can you explain to me where this number comes from? I'm wondering if there is a way to automate the step of getting the submission ID from the VolWQDB. Particularly so that we don't accidentally end up with duplicate IDs.